mezzanine/accounts/admin.py

```diff
@@ -45,20 +46,15 @@ def save_model(self, request, obj, form, change):
             send_verification_mail(request, user, "signup_verify")
 
 
-try:
-    class ProfileInline(admin.StackedInline):
-        model = get_profile_model()
-        can_delete = False
-        template = "admin/profile_inline.html"
-        extra = 0
+class ProfileInline(admin.StackedInline):
+    model = get_profile_model()
+    can_delete = False
+    template = "admin/profile_inline.html"
+    extra = 0
 
-        def get_min_num(self, request, obj=None, **kwargs):
-            """This causes profile forms to be shown when editing but hidden
-            when creating. If min_num is fixed at 1, Django's initial user
-            creation form fails if the profile model has a required field."""
-            return 0 if obj is None else 1
 
-    UserProfileAdmin.inlines += (ProfileInline,)
+try:
+    UserProfileAdmin.inlines += (get_profile_inline_form(),)
```

Comment:
Agree with @jerivas, function seems unnecessary, especially since the default should never fail since the setting will default to something that always works